src/openrct2/drawing/Drawing.h

```diff
@@ -296,6 +296,33 @@ enum class FilterPaletteID : int32_t
     PaletteGlassDarkPink = PaletteGlass + COLOUR_DARK_PINK,
     PaletteGlassBrightPink = PaletteGlass + COLOUR_BRIGHT_PINK,
     PaletteGlassLightPink = PaletteGlass + COLOUR_LIGHT_PINK,
+    PaletteG2 = 144,
+    PaletteG2Invisible = 166,
```

Comment:
G2 should not be mentioned here it means nothing and is an implementation detail.